org.eclipse.jdt.core.compiler.batch/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,32 @@
+Manifest-Version: 1.0
+Main-Class: org.eclipse.jdt.internal.compiler.batch.Main
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse Compiler for Java(TM)
+Bundle-SymbolicName: org.eclipse.jdt.core.compiler.batch
+Bundle-Version: 3.32.0.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: Eclipse.org
+Automatic-Module-Name: org.eclipse.jdt.core.compiler.batch
+Bundle-ActivationPolicy: lazy
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Export-Package: org.eclipse.jdt.core,
```

Comment:
I think at least the public packages should get a version, (e.g. 1.0.0) and include a use-clause (you can calculate this on the export package tab).

Reply:
Let discuss that later, to not lost the main focus. AFAIK this is unusual for Eclipse bundles to version packages **additionally** to the bundle version. Note: the bundle **is not new** (it was just constructed"on the fly by ant"), it is already published under same id, so version is not 1.0.0 anyway.

Reply:
This is not additionally. Currently the package is completely _unversioned_ that means every consumer must use require bundle, see for example: 

https://docs.osgi.org/whitepaper/semantic-versioning/010-executive-summary.html

so it seems a good opportunity to start with that.

Reply:
> it seems a good opportunity to start with that

Let us first get that thing working, OK? After that feel free to propose any improvements. I just don't want to loose focus here and don't have extra time anyway. My goal is to do whatever is **needed** first, ideally in 4.27 M1.

Reply:
Sure, I just found that such things often help to find the culprit, but we can leave that out for sure as I have no idea why it should make a difference.

Reply:
The problem here is that we have split packages / fragments / whatsoever dependencies with existing jdt core code that might cause unexpected side effects. Create a task if you want to follow up.